lib/liquid/parse_context.rb

```diff
@@ -0,0 +1,39 @@
+module Liquid
+  class ParseContext
```

Comment:
definitely don't use method_missing


- tobi
CEO Shopify

On Mon, Jul 6, 2015 at 9:40 AM, Tristan Hume <notifications@github.com>
wrote:

> In lib/liquid/parse_context.rb
> <https://github.com/Shopify/liquid/pull/614#discussion_r33934235>:
>
> > @@ -0,0 +1,39 @@
> > +module Liquid
> > +  class ParseContext
>
> I'm tempted to make this delegate method_missing to the @options hash so
> that it maintains full API compatibility with old tags. But that is hacky
> and I'm not sure this will actually break any tags given that it does have
> [].
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/liquid/pull/614/files#r33934235>.
>